ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -39,16 +41,38 @@ object LedgerApiTestTool {
   private def exitCode(summaries: Vector[LedgerTestSummary], expectFailure: Boolean): Int =
     if (summaries.exists(_.result.isLeft) == expectFailure) 0 else 1
 
-  private def printAvailableTests(): Unit = {
-    println("Tests marked with * are run by default.")
+  private def printAvailableTests(config: Config): Unit = {
+    val session = new LedgerSession(
```

Comment:
If I'm not mistaken invoking this constructor uses the configuration to actually start to initiate a connection with the participant to the ledger under test. If that's the case, this would prevent `--list` from working without having a running participant and without providing its address. Can you double check whether that's the case? If that's so, we should find some other way to list all test cases. The easiest way that comes to mind is probably having a companion object for each suite with the names of the tests, and somehow use that, but I'm not sure about the feasibility.

Reply:
```
java -jar bazel-bin/ledger/ledger-api-test-tool/ledger-api-test-tool_deploy.jar --list
```
works for me without having a sandbox running. I don't think any connection is actually established before `.run` is called on the test suite.

Reply:
Ok, good to know, thanks.